### PR TITLE
Issue 3348

### PR DESCRIPTION
**Pull request by milovanderlinden**
## References

#3348
## Objectives

Fix a path error when running consul in http mode behind a https proxy

## Visual Changes
none

### CODE REVIEW

**Comment by milovanderlinden** [comment]
Oops. I see that somehow two PR are getting merged here. Please invalidate if necessary, I can re-create.

**Comment by milovanderlinden** [comment]
Ok, the branch has been corrected and now it only contains this particular fix.

File: app/assets/stylesheets/layout.scss [hunk 1]
```diff
@@ -1583,6 +1584,22 @@ table {
     top: 0;
   }
 }
+.button.button-google_oauth2 {
```
[the remark this answers is not in the document]
Comment (milovanderlinden) [reply app/assets/stylesheets/layout.scss]: Hound, you have not picked up the new change I made. I am sorry, but I can only close this conversation.

**Comment by javierm** [comment]
The related issue was closed by #3766.

**Pull request closed without merging**